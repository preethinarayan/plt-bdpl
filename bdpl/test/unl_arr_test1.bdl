bit[*] a (fieldsize 20);
file "./test.dat" inp_file;
print("finished reading input file ... creating array");
read(inp_file,a);
print(a);
