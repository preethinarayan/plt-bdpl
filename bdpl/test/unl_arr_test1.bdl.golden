x:\test\aa.mp3.mp3 (The system cannot find the file specified)
